Source/RapyutaSimulationPlugins/Private/Tools/OccupancyMapGenerator.cpp

```diff
@@ -62,7 +62,7 @@ void AOccupancyMapGenerator::BeginPlay()
     }
 
     // write to file
-    check(WriteToFile(NCellsX, NCellsY, Origin.X / 100.f, -(Center.Y + Extent.Y) / 100.f));
+    WriteToFile(NCellsX, NCellsY, Origin.X / 100.f, -(Center.Y + Extent.Y) / 100.f);

```

Comment:
Some error log upon failure?

Reply:
Currently the function return always false

Reply:
always [true](https://github.com/rapyuta-robotics/RapyutaSimulationPlugins/blob/devel/Source/RapyutaSimulationPlugins/Private/Tools/OccupancyMapGenerator.cpp#L90)? So may need to refactor it a bit to reflect the return of `FFileHelper::SaveArrayToFile()` I guess